webapp/.eslintrc.json

```diff
@@ -32,7 +32,10 @@
     "beforeEach": true
   },
   "settings": {
-    "import/resolver": "webpack"
+    "import/resolver": "webpack",
+    "react": {
+      "version": "detect"
```

Comment:
You'd think "detect" would be default :)

Reply:
Probably is the default, but unless you put it here, it will complain about it not being set.